content/docs/deployment/csmoperator/_index.md

```diff
@@ -170,13 +171,10 @@ Once the driver `Custom Resource (CR)` is created, you can verify the installati
 
 *  Check if ContainerStorageModule CR is created successfully using the command below:
     ```
-    $ kubectl get csm -n <driver-namespace>
+    $ kubectl get csm/<name-of-custom-resource> -n <driver-namespace> -o yaml
     ```
-* Check the status of the CR to verify if the driver installation is successful.
+* Check the status of the CR to verify if the driver installation is in the `Succeeded` state. If the status is not `Succeeded`, see the [Troubleshooting guide](./troubleshooting/) for more information.
```

Comment:
@tdawe : This link will not work with Hugo

Reply:
@shanmydell it worked on my Hugo instance version `hugo v0.89.0-DEV+extended`. Is there another version being used for our live site?

Reply:
@tdawe : It ll goto the page , but not the section "troubleshooting/#my-dell-csi-driver-install-failed-how-do-i-fix-it"

Reply:
Ok right, updated to go to proper section